cd/update_app_version.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+# Gets project root directory by calling two nested "dirname" commands on the this file 
+getRootDirectory() {
+    echo "$( cd $(dirname $(dirname $0)) > /dev/null 2>&1 && pwd )"
+}
+
+# Get version in app_version
+getAppVersion() {
+    root_dir="$(getRootDirectory)"
+    echo "$(cat ${root_dir}/app_version)"
+}
+
+# Get Command name ("update_app_version.sh")
+getCommand() {
+    echo "$(basename $0)"
+}
+
+# Increments the bugfix version by 1
+createNewVersion() {
+    old_version="$1"
+    major_and_minor="$(echo ${old_version} | sed 's|^\(.*\.\).*|\1|')"
+    bugfix="$(echo ${old_version} | sed 's|.*\.||')"
+    let "bugfix++"
+    echo "${major_and_minor}""${bugfix}"
+}
+
+# Performs a Find and replace on the Helm charts and the app_version file
+findAndReplace() {
+    find "$(getRootDirectory)" \( -iname Chart.y*ml -o -name app_version \) -exec sed -i'' -e "s:$1:$2:g" {} +
```

Comment:
Shouldn't be a problem most of the time, but if we ever have a repo/app version which matches the version of the ZooKeeper chart (currently 2.1.1) that we have as a dependency to the Gaffer chart, then this command will update that version number too.

Reply:
As you say, this shouldn't be a problem for now (as long as we're on a 0.x release and all dependencies are on a >0.x release). I'm not sure it's worth fixing now but worth raising an issue to fix in future.

Reply:
👍